examples/compiled/waterfall_chart.vg.json

```diff
@@ -30,12 +30,10 @@
       "source": "source_0",
       "transform": [
         {
-          "type": "window",
-          "params": [null],
+          "type": "joinaggregate",
```

Comment:
1) Don't rename this file

2) waterfall should use window (with default frame = `[null, 0]`)

Reply:
Waterfall chart is still wrong.  It should not use `joinaggregate`.

Reply:
![image](https://user-images.githubusercontent.com/111269/53119459-6ad50f00-3504-11e9-8f4c-540bbb10b13a.png)